Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/CvarAdapter.cpp

```diff
@@ -102,12 +111,14 @@ namespace AZ::DocumentPropertyEditor
                     {
                         VectorType newContainer = Dom::Utils::ValueToType<VectorType>(value).value_or(VectorType());
                         // ConsoleCommandContainer holds string_views, so ensure we allocate our parameters here
-                        AZStd::fixed_vector<CVarFixedString, ElementCount> newValue;
+                        AZStd::fixed_vector<CVarFixedString, ElementCount + 1> newValue;
+                        newValue.push_back(functor->GetName());
                         for (int i = 0; i < ElementCount; ++i)
                         {
                             newValue.push_back(ConsoleTypeHelpers::ToString(newContainer.GetElement(i)));
                         }
-                        (*functor)(ConsoleCommandContainer(AZStd::from_range, newValue));
+                        AZ::Interface<AZ::IConsole>::Get()->PerformCommand(ConsoleCommandContainer(AZStd::from_range, newValue));
+
```

Comment:
This can also be called with the command parameter separated from the Console Command Container as follows.
```suggestion
                        AZ::Interface<AZ::IConsole>::Get()->PerformCommand(functor->GetName(), ConsoleCommandContainer(AZStd::from_range, newValue));

```

Reply:
yep, fixed thanks!